app/models/pawn.rb

```diff
@@ -1,2 +1,67 @@
 class Pawn < Piece
+
+  def valid_move?(destination_x, destination_y)
+    #invalid move error raises if moving to same current
+    #xy coordinates
+
+    #check if it's backward/horizontal move
+    invalid_move = invalid_move?(destination_y)
+    return false if invalid_move
+
+    #find distance
+    y_coords = current_to_destination_y_coordinates(destination_y)
+    y_distance = y_coords.length
+
+    #check if pawn can move correct # of spaces
+    if first_move?
+      #check if path is obstructed
+      obstructed = is_obstructed?(destination_x, destination_y)
+      return false if obstructed
+      valid_spaces = (y_distance <= 1 ? true : false)
```

Comment:
This code
`(y_distance <= 1 ? true : false)`

Could be simplified to:
`(y_distance <= 1)`

Will result in the same thing :)